services/director-v2/src/simcore_service_director_v2/api/routes/services.py

```diff
@@ -63,3 +70,29 @@ async def get_service_versioned(
     forward_request: Response = Depends(forward_to_director_v0),
 ):
     return forward_request
+
+
+@router.post(
+    "/{service_key:path}/{service_version}/dynamic-sidecar:require",
+    summary="returns True if service must be ran via dynamic-sidecar",
+)
+async def requires_dynamic_sidecar(
+    service_key: str = constr(regex=KEY_RE, strip_whitespace=True),
+    service_version: str = ServiceKeyVersionPath,
+    director_v0_client: DirectorV0Client = Depends(get_director_v0_client),
+) -> bool:
+    decoded_service_key = urllib.parse.unquote_plus(service_key)
+
+    # check the type of service, if not dynamic do
+    # not fetch the labels
+    # (simcore)/(services)/(comp|dynamic|frontend)
+    service_type = decoded_service_key.split("/")[2]
+    if service_type != "dynamic":
+        return False
+
+    simcore_service_labels: SimcoreServiceLabels = (
```

Comment:
Uhm... I think this is OK like this. It is skipping calls to director-v0 if the service is not dynamic.
Since the workbench can be big, thought this should be better. This would be the point where the errors get catched and handled.
The main benefit is to avoid calls to director-v0 and the docker engine which could be skipped.